Invoice.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+
+namespace VisualCode.Entities
+{
+    public class Invoice
```

Comment:
You started very well, using plural for the name of class, but after not, for entities I think is better use plural for name o class.